turbinia/workers/analysis/windows_acct.py

```diff
@@ -74,6 +74,11 @@ def run(self, evidence, result):
           status='Unable to extract hashes from registry files: {0:s}'.format(
               str(exception)))
       return result
+    if os.path.isfile(os.path.join(location, 'Windows', 'NTDS', 'ntds.dit')):
+      (adcreds, adhashnames) = self._extract_ad_hashes(result, location)
```

Comment:
Can we wrap this in a try/catch block for TurbiniaException (similar to above when `_extract_windows_hashes()` is called) so we can close out `result` and set a status?  Also, should we try both extraction methods (windows hashes and AD creds) and only completely fail if both of those result in errors?

Reply:
It's my understanding that all Windows machines should have local creds in the registry, but only DCs will have stuff in the AD datastore. Thus we can expect this method to fail a reasonable percentage of the time. We also cannot extract the AD creds if the registry extraction fails since it uses one of the hives.

Reply:
Ok, makes sense, thanks.